src/core/render/tpl.js

```diff
@@ -91,7 +93,9 @@ export function tree(toc, tpl = '<ul class="app-sub-sidebar">{inner}</ul>') {
 
   let innerHTML = '';
   toc.forEach(node => {
-    innerHTML += `<li><a class="section-link" href="${node.slug}" title="${node.title}">${node.title}</a></li>`;
+    innerHTML += `<li><a class="section-link" href="${
+      node.slug
+    }" title="${escapeHtml(node.title)}">${node.title}</a></li>`;
```

Comment:
Just one optional thought

 How about escaping `node.title` ?

Reply:
I didn't really understand the meaning of the question. the problem can be seen here https://posthtml.org/#/packages on sidebar '">'

Reply:
fixed via #1336